ROADMAP.md

```diff
@@ -8,21 +8,25 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 See the [GitHub project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
-## Rook Ceph 1.9
+## Rook Ceph 1.10
 
-The following high level features are targeted for Rook v1.9 (early April 2022). For more detailed project tracking see the [v1.9 board](https://github.com/rook/rook/projects/24).
+The following high level features are targeted for Rook v1.10 (early August 2022). For more detailed project tracking see the [v1.10 board](https://github.com/rook/rook/projects/26).
+
+* Remove support for Ceph Octopus (support remains for Pacific and Quincy) [#10338](https://github.com/rook/rook/issues/10338)
+* Create tool to analyze cluster health and advise on resolving common health issues [Krew #32](https://github.com/rook/kubectl-rook-ceph/issues/32)
+* Check for existing subvolumes before allowing a filesystem to be uninstalled [#9915](https://github.com/rook/rook/pull/9915)
+* Use more specific cephx accounts to better differentiate the source of configuration changes [#10169](https://github.com/rook/rook/issues/10169)
+* Automate node fencing for application failover in some scenarios [#1507](https://github.com/rook/rook/issues/1507)
+* Object Store
+  * Service account authentication with the Vault agent [#9872](https://github.com/rook/rook/pull/9872)
+  * Support for AWS Server Side Encryption (SSE) [#10318](https://github.com/rook/rook/pull/10318)
```

Comment:
I am planning to include some improvements for rgw multisite in 1.10 like https://github.com/rook/rook/pull/10323